test/kind-config/noCNI.yaml

```
@@ -0,0 +1,10 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+
+networking:
+  disableDefaultCNI: true
+  podSubnet: 192.168.0.0/16
+
```

```suggestion
  apiServerPort: 6443
```
Setting this to something constant helps with testing and tunneling on ec2.